test/Makefile.am

```diff
@@ -2,7 +2,9 @@
 # Makefile.am for autoinstallation/test
 #
 
-TESTS = AutoInstallRules_test.rb
+TESTS = \
```

Comment:
Interesting. I use `rake test:unit` to run them and they work. It will be interesting to see what's the problem (but better we could discuss it outside of this PR).